visualization/app/codeCharta/util/fileNameHelper.ts

```diff
@@ -13,23 +15,22 @@ export class FileNameHelper {
 	}
 
 	private static getFileNameWithoutTimestamp(fileName: string, isDeltaState: boolean) {
-		const dateRegex = /_\d{4}(?:-\d{1,2}){2}_\d{1,2}-\d{1,2}\./
-
 		if (!isDeltaState) {
-			if (dateRegex.test(fileName)) {
-				return fileName.slice(0, Math.max(0, dateRegex.exec(fileName).index))
+			const dateMatch = dateRegex.exec(fileName)
+			if (dateMatch) {
+				return fileName.slice(0, dateMatch.index)
 			}
-			if (fileName.includes(CodeChartaService.CC_FILE_EXTENSION)) {
-				return fileName.slice(0, Math.max(0, fileName.search(CodeChartaService.CC_FILE_EXTENSION)))
+			if (fileName.endsWith(CodeChartaService.CC_FILE_EXTENSION)) {
+				return fileName.slice(0, -CodeChartaService.CC_FILE_EXTENSION.length)
```

Comment:
This is a bug fix since it now does not match `.cc` anywhere in the middle of a filename anymore.